packages/components/src/templates/next/layouts/Collection/utils.ts

```diff
@@ -24,12 +30,20 @@ export const getAvailableFilters = (
       } else {
         years[year] = 1
       }
+    } else {
+      numberOfUndefinedDates += 1
     }
   })
 
+  const yearFilterItems = Object.entries(years).map(([label, count]) => ({
+    id: label.toLowerCase(),
+    label,
+    count,
+  }))
+
   const availableFilters: FilterType[] = [
     {
-      id: "category",
+      id: FILTER_ID_CATEGORY,
       label: "Category",
       items: Object.entries(categories).map(([label, count]) => ({
```

Comment:
nit: btw I think we should sort the labels by ascending alphabetical order

Reply:
or should it be based on "count" descreasing? since it's likely more relevant?